examples/monitoring/task_classification/README.md

```diff
@@ -0,0 +1,210 @@
+
+```python
+import numpy as np
+
+import bentoml
+from bentoml.io import NumpyNdarray
+
+CLASS_NAMES = ["setosa", "versicolor", "virginica"]
+
+iris_clf_runner = bentoml.sklearn.get("iris_clf:latest").to_runner()
+svc = bentoml.Service("iris_classifier", runners=[iris_clf_runner])
+
+
+@svc.api(
+    input=NumpyNdarray.from_sample(np.array([[4.9, 3.0, 1.4, 0.2]], dtype=np.double)),
+    output=NumpyNdarray(),
+)
+async def classify(input_series: np.ndarray) -> np.ndarray:
+    with bentoml.monitor("iris_classifier_prediction") as mon:
+        mon.log(
+            data=input_series[0][0],
+            name="sepal length",
+            role="feature",
+            data_type="numerical",
+        )
+        mon.log(
+            input_series[0][1],
+            name="sepal width",
+            role="feature",
+            data_type="numerical",
+        )
+        mon.log(
+            input_series[0][2],
+            name="petal length",
+            role="feature",
+            data_type="numerical",
+        )
+        mon.log(
+            input_series[0][3],
+            name="petal width",
+            role="feature",
+            data_type="numerical",
+        )
+        result = await iris_clf_runner.predict.async_run(input_series)
+        mon.log(
+            CLASS_NAMES[result[0]],
+            name="pred",
+            role="prediction",
+            data_type="categorical",
+        )
+    return result
+```
+
+```bash
+bentoml serve service.py:svc --reload
+```
+
+Open your web browser at http://127.0.0.1:3000 to view the Bento UI for sending test requests.
+
+You may also send request with `curl` command or any HTTP client, e.g.:
+
+```bash
+curl -X POST -H "content-type: application/json" --data "[[5.9, 3, 5.1, 1.8]]" http://127.0.0.1:3000/classify
+```
+
+
+Then you can find the exported data under the `./monitoring/<monitor_name>/data` directory.
+Here's the example output:
+
+```json
+{"timestamp": "2022-11-02T12:38:38.701396", "request_id": 8781503815303167270, "sepal length": 5.9, "sepal width": 3.0, "petal length": 1.4, "petal width": 0.2, "pred": "0"}
+{"timestamp": "2022-11-02T12:38:48.345552", "request_id": 14419506828678509143, "sepal length": 4.9, "sepal width": 3.0, "petal length": 1.4, "petal width": 0.2, "pred": "0"}
+```
+
+
+### Customizing the monitoring
+
+You can customize the monitoring by modifying the config file of bentoml. The default is:
+
+```yaml
+monitoring:
+  enabled: true
+  type: default
+  options:
+    output_dir: ./monitoring
+```
+
+You can draft your own bentoml config file `deployment.yaml` and change the `output_dir` to any directory you want. You can also use other monitoring solutions by changing the `type` to your desired handler. For example, if you want to use the `arize` handler, you can change the config to:
+
+```yaml
+monitoring:
+  enabled: true
+  type: bentoml_plugins.arize.ArizeMonitor
```

Comment:
If this is not an arize example, then this shouldn't be specified like this. Can you use a mock example instead?